src/Bullet/App.php

```diff
@@ -59,7 +59,8 @@ function($response) {
             function($response) {
                 $response->contentType('application/json');
                 $response->content(json_encode($response->content()));
-            }
+            },
+            'array-content'
```

Comment:
I'd like to name this `array_json` since it covers both the content input, and the resulting output.